extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1019,6 +1047,23 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     return super.unlayoutCallback();
   }
 
+  /** @override */
+  getSafeframePath() {
+    const randomSubdomainExperimentBranch =
+      RANDOM_SUBDOMAIN_SAFEFRAME_BRANCHES.EXPERIMENT;
+    if (!this.experimentIds.includes(randomSubdomainExperimentBranch)) {
+      return super.getSafeframePath();
+    }
+    if (!this.safeFrameRandomSubdomain_) {
```

Comment:
this.safeFrameRandomSubdomain_ = this.safeFrameRandomSubdomain_ || this.getRandomString_();

Reply:
done